tox.ini

```diff
@@ -81,15 +81,21 @@ commands =
          {toxinidir}/tests/docker/scripts/pylint.sh python/nav --jobs=4 --rcfile=python/pylint.rc --disable=I,similarities --load-plugins pylint_django --output-format=parseable
 
 [testenv:docs]
+basepython = python3.5
 description = Just build the Sphinx documentation
 deps = pip-tools
 setenv =
     PYTHONPATH = {toxinidir}/python:{toxinidir}/tests
     DJANGO_SETTINGS_MODULE = nav.django.settings
     LC_ALL=C.UTF-8
     LANG=C.UTF-8
-whitelist_externals = /bin/sh
+whitelist_externals =
+    /bin/sh
+    touch
+# The `touch` work arounds a bug pip sometimes has with setuptools. Symptom:
+# UninstallationError: Cannot remove entries from nonexistent file <path>/easy-install.pth
 commands =
+         touch {envdir}/lib/python3.5/site-packages/easy-install.pth
```

Comment:
I'm thinking this may actually be superfluous thanks to 85820069b4ae66ca899ca3f486c553be856e6ebc

It would be interesting to chop this commit off and see if test suite still passes.